tests/async/conftest.py

```diff
@@ -17,6 +17,12 @@
 from playwright import async_playwright
 
 
+# Will mark all the tests as async
+def pytest_collection_modifyitems(items):
+    for item in items:
+        item.add_marker(pytest.mark.asyncio)
+
```

Comment:
Can you explain to me why this is needed? (I'm new to async in Python.)

Reply:
Before it was in the global `conftest.py`. A `conftest.py` will be executed through the directory three and by that they can overwrite each other (which is nearest to the actual test).

This specific piece added asyncio support to the tests which was before available in all the tests. But since we split async and sync I limited the scope of it.

Reply:
Thanks! So async tests need to be explicitly (or programmatically) marked as async for pytest to run them?

Reply:
exactly with: https://github.com/pytest-dev/pytest-asyncio

You can do it manually with the decorator per test or in our case just for all the tests in that directory.